service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -101,20 +125,17 @@ set_guest_attributes "${STATUS_ATTRIBUTE}" "STARTED"
 # Install nbstripout for the jupyter user in all git repositories.
 sudo -u "${JUPYTER_USER}" sh -c "/opt/conda/bin/nbstripout --install --global"
 
-# Install Nextflow. Use an edge release that allows overriding the default compute engine SA and VPC network
-export NXF_VER=21.05.0-edge
```

Comment:
I _think_ you need to keep these variables. Later on when you run `"curl -s https://get.nextflow.io | bash"`, that nextflow install script reads these variables.

Reply:
@emmarogge tested Nextflow with a VM created using this PR's post-startup.sh